### PR TITLE
Update skills.yml

### PR DESCRIPTION
**Pull request by elwyncrestha**
* Added two skills

### CODE REVIEW

**Comment by elwyncrestha** [comment]
It's not mandatory I guess. I am not a bot though haha.

**Comment by elwyncrestha** [comment]
@xavigh did you check your pull request, are you able to merge the request now?

File: _data/skills.yml [hunk 1]
```diff
@@ -6,4 +6,12 @@
 
 # Leadership
 - skill: Leadership
-  description: Managed multiple asynchronous teams in the development, maintenance, and release of various web applications and websites.
\ No newline at end of file
+  description: Managed multiple asynchronous teams in the development, maintenance, and release of various web applications and websites.
+  
+# Skill1
```
Comment (xavigh) [inline _data/skills.yml]: for example add , I play Piano and can read musics sheet

**Comment by xavigh** [comment]
> @xavigh did you check your pull request, are you able to merge the request now?

I will try now, thank you !

**Comment by elwyncrestha** [comment]
__Thanks!__

**Comment by elwyncrestha** [comment]
But I forgot what we were supposed to do haha

**Comment by xavigh** [comment]
> But I forgot what we were supposed to do haha

https://lab.github.com/public/managing-merge-conflicts.pdf

**Comment by elwyncrestha** [comment]
@xavigh are you able to solve the problem now?

**Comment by xavigh** [comment]
> @xavigh are you able to solve the problem now?

Yes I can progress now, I have merged branches and deleted the branch created as it´s alredy merged.
any problems we can try to help each other, this is all new to me, trying to get a job as a front end developer and need to do all this thing about GitHub... what about you? what are you up to?

**Comment by elwyncrestha** [comment]
> Yes I can progress now, I have merged branches and deleted the branch created as it´s alredy merged.
> any problems we can try to help each other, this is all new to me, trying to get a job as a front end developer and need to do all this thing about GitHub... what about you? what are you up to?

You mean you can progress the course in your __Learning Lab__? I'm still stuck in the first step.
And about me, I'm working as an intern here in Nepal. I do backend in JAVA Web. I'm still learning lots of things.

**Comment by elwyncrestha** [comment]
I know the process of resolving the merge conflict, but I was just trying to complete the course in the Learning Lab, and I'm stuck here.

**Pull request merged**